src/Model/Behavior/DuplicatableBehavior.php

```diff
@@ -35,7 +35,8 @@ class DuplicatableBehavior extends Behavior
         'set' => [],
         'prepend' => [],
         'append' => [],
-        'saveOptions' => []
+        'saveOptions' => [],
+        'preserveKeyFromCompositeKey' => ''
```

Comment:
Why is this this a string, what if composite key consistents of 3 fields and 2 need to be removed?

Reply:
Done